src/assets/wise5/classroomMonitor/dataExport/strategies/AbstractDataExportStrategy.ts

```diff
@@ -162,4 +163,8 @@ export abstract class AbstractDataExportStrategy implements DataExportStrategy {
       `${componentNumber}_${componentTypeWithUnderscore}_work.csv`
     );
   }
+
+  generateCSVFile(rows: any[], fileName: string): void {
+    generateCSVFile(rows, fileName);
+  }
```

Comment:
Should we call generateCSVFile() directly and not add this function?

Reply:
This is required for testing since I couldn't find an easy way to monitor the parameters that are sent to generateCSVFile() if we use the function directly from csv.ts. If we could find a better solution to this in the future, that would be good.

Reply:
Ok, let's refactor this in the future. It's not ideal to have to write extra code just to make the tests pass.